File: NUnitProject/NUnitProject/Tests/LoginTests.cs
using NUnit.Framework;
using NUnitProject.Pages;
using NUnitProject.Tests.Base;
using System;
using System.Collections.Generic;
using System.Diagnostics;
using System.Threading;

namespace NUnitProject.Tests
{
    public class LoginTests : BaseTest
    {
        public LoginTests()
        {
            Logger.Instance.Info("LoginTests Thread-id: " + Thread.CurrentThread.ManagedThreadId);
        }

        [TestCaseSource("DataProviderLogin")]
        public void TestSuccessfulLogin(String user, String pass)
        {
            Logger.Instance.Info("testSuccessfulLogin Thread-id: " + Thread.CurrentThread.ManagedThreadId);

            LoginPage loginPage = new LoginPage(WebDriver);
            EmployeePage employeePage = loginPage.LoginAs(user, pass);
            Assert.IsTrue(employeePage.IsEmployeePageDisplayed());
            Assert.AreEqual(employeePage.GetUserNameText(), user);
        }

        static IEnumerable<object> DataProviderLogin()
        {
            Logger.Instance.Info("testSuccessfulLogin data-provider Thread-id: " + Thread.CurrentThread.ManagedThreadId);
            yield return new object[] { "admin", "admin123" };
            yield return new object[] { "admin", "admin123" };
        }
    }
}

File: JUnitProject/src/test/java/login/LoginTests.java
package login;

import base.BaseTest;
import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.params.ParameterizedTest;
import org.junit.jupiter.params.provider.CsvSource;
import pages.EmployeePage;
import pages.LoginPage;

public class LoginTests extends BaseTest {

    public LoginTests(){
        System.out.println("LoginTests Thread-id: " + Thread.currentThread().getId());
    }

    @ParameterizedTest
    @CsvSource({ "admin,admin123", "admin,admin123" })
    public void testSuccessfulLogin(String user, String pass){
        System.out.println("testSuccessfulLogin Thread-id: " + Thread.currentThread().getId());

        LoginPage loginPage = new LoginPage(webDriver);
        EmployeePage employeePage = loginPage.loginAs(user,pass);
        Assertions.assertTrue(employeePage.isEmployeePageDisplayed());
        Assertions.assertEquals(employeePage.getUserNameText(), user);
    }
}
